packages/server/src/schema/uuid/abstract-entity/resolvers.ts

```diff
@@ -48,6 +56,64 @@ export const resolvers: InterfaceResolvers<'AbstractEntity'> &
       return entityRevision.__typename
     },
   },
+  EntityQuery: {
+    async deletedEntities(_parent, payload, { dataSources }) {
+      const LIMIT = 1000
+      const { first = 100, after, instance } = payload
+
+      if (first > LIMIT)
+        throw new UserInputError(`'first' may not be higher than ${LIMIT}`)
+
+      const deletedAfter = after
+        ? (
+            JSON.parse(Buffer.from(after, 'base64').toString()) as {
```

Comment:
```suggestion
            JSON.parse(Buffer.from(after, 'base64').toString())
```

never use `as` for data coming from the outside. use `t.type({ dateOfDeletion: ... }).is()`